### PR TITLE
Fix metadata

### PR DESCRIPTION
**Pull request by PatoBeltran**
## Bug
Fixes: https://github.com/NuGet/Home/issues/6518

I made sure that all of the nupkgs we ship have the following correct metadata:

* **Author**: Microsoft
*  **Copyright**: &#169; Microsoft Corporation. All rights reserved.
* **projectUrl**: https://aka.ms/nugetprj
* **licenseUrl**: https://aka.ms/nugetlicense
* **Description**: `<package description>`. Microsoft holds the copyright for this NuGet package.
    .NET Foundation holds the copyright to the source.

//cc. @rrelyea

### CODE REVIEW

File: src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/NuGet.SolutionRestoreManager.Interop.csproj [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
     <authors>Microsoft</authors>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/NuGet/NuGet.Client/dev/LICENSE.txt</PackageLicenseUrl>
-    <PackageProjectUrl	>https://github.com/NuGet/NuGet.Client</PackageProjectUrl	>
+    <PackageLicenseUrl>https://aka.ms/nugetlicense</PackageLicenseUrl>
+    <PackageProjectUrl>https://aka.ms/nugetprj</PackageProjectUrl>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <AssemblyDescription>APIs for invoking NuGet Restore Manager in Visual Studio.</AssemblyDescription>
-    <Description>APIs for invoking NuGet Restore Manager in Visual Studio.</Description>
-    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <AssemblyDescription>APIs for invoking NuGet Restore Manager in Visual Studio. Microsoft holds the copyright for this NuGet package. .NET Foundation holds the copyright to the source.</AssemblyDescription>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/NuGet.SolutionRestoreManager.Interop.csproj]: I don't get it. 
Why should the description contain copyright information? 
What's the copyright field for then?

File: src/NuGet.Clients/NuGet.VisualStudio/NuGet.VisualStudio.csproj [hunk 2]
```diff
@@ -11,16 +11,16 @@
 
   <PropertyGroup>
     <PackageId>NuGet.VisualStudio</PackageId>
-    <AssemblyDescription>APIs for invoking NuGet services in Visual Studio.</AssemblyDescription>
-    <Description>APIs for invoking NuGet services in Visual Studio.</Description>
+    <AssemblyDescription>APIs for invoking NuGet services in Visual Studio. Microsoft holds the copyright for this NuGet package. .NET Foundation holds the copyright to the source.</AssemblyDescription>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.VisualStudio/NuGet.VisualStudio.csproj]: Same here. Assembly description is a misuse for copyright claims.

**Review by nkolev92 — CHANGES_REQUESTED** [review]
The description metadata is misused.

File: src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/NuGet.SolutionRestoreManager.Interop.csproj (hunk 1 above)
Comment (PatoBeltran) [reply src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/NuGet.SolutionRestoreManager.Interop.csproj]: @karann-msft  https://github.com/NuGet/Home/issues/6518#issuecomment-408205989

**Comment by PatoBeltran** [comment]
@nkolev92 @rrelyea Addressed comments, ready for review

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**